### PR TITLE
Add uninstall instructions.

### LINKED ISSUE #299: Add uninstall instructions
From: https://usfirst.collab.net/sf/discussion/do/listPosts/projects.frc2020beta/discussion.task_reports.topc1998?pageSize=-1#post_post6355

Should delete wpilib/YEAR directory (or frc2019 directory), delete shortcuts, delete environment variables.

### PR DESCRIPTION
**Pull request by sciencewhiz**
Fixes #299

### CODE REVIEW

**Comment by AustinShalit** [comment]
I think we need to add instructions about uninstalling the WPILib VSIX from VS Code.

File: source/docs/getting-started/getting-started-frc-control-system/wpilib-setup.rst [hunk 1]
```diff
@@ -261,3 +261,33 @@ WPILib Installation Guide
...
+WPILib is designed to install to different folders for different years, so that it is not necessary to uninstall a previous version before installing this year's WPILib. However, the following instructions can be used to uninstall WPILib if desired.
+
+.. tabs::
+
+  .. tab:: Windows
+
+    1. Delete the appropriate wpilib folder (2019: ``c:\Users\Public\frc2019``, 2020 and later: ``c:\Users\Public\wpilib\YYYY`` where ``YYYY`` is the year to uninstall)
+    2. Delete the desktop icons at ``C:\Users\Public\Public Desktop``
+    3. Delete the path environment variables. 
+      1. In the start menu, type environment and select "edit the system environment variables"
```
Comment (Daltz333) [inline source/docs/getting-started/getting-started-frc-control-system/wpilib-setup.rst]: Alignment should probably be three spaces

**Comment by sciencewhiz** [comment]
> I think we need to add instructions about uninstalling the WPILib VSIX from VS Code.

Instead of uninstalling VS Code, or in addition?

**Comment by AustinShalit** [comment]
I think on Windows the instructions are actually ok to recommend uninstalling VS Code because the installer installs it. But in the Mac and Linux instructions the user manually installs VS Code so maybe we need to be more fine grained?

**Comment by AustinShalit** [comment]
The brew cask that I have for Mac only uninstalls the VSIX and leaves VS Code installed.

**Pull request merged**